meetup.rb

```diff
@@ -0,0 +1,72 @@
+require 'httparty'
+require 'fileutils'
+require 'colorize'
+require 'dotenv'
+
+# .env
+Dotenv.load
+meetup_api_key = ENV["MEETUP_API_KEY"]
+
+meetup_group_url = "ny-tech"
+# meetup_group_url = "RubyFTW-Fort-Worth-Ruby-User-Group"
+base_uri = "https://api.meetup.com/2/events?key=#{meetup_api_key}&group_urlname=#{meetup_group_url}&sign=true"
+response = HTTParty.get(base_uri)
+body = JSON.parse(response.body)
+
+mu = body["results"][0]
+
+@file_name_date = Time.at(mu["time"]/1000).strftime("%Y-%m-%d")
+@title = mu["name"]
+@description = mu["description"]
+@long_date = Time.at(mu["time"]/1000).strftime("%B %d, %Y")
+@link = mu["event_url"]
+@time = Time.at(mu["time"]/1000).strftime("%I:%M%P")
+@location = mu["venue"]["name"]
+@location_url = "https://www.google.com/maps/search/#{@location}"
+@rsvp_url = mu["event_url"]
+
+@directory = "_upcoming_events"
+@archived_directory = "_archived_events"
+@directory_file_count = Dir[File.join(@directory, '**', '*')].count { |file| File.file?(file) }
+@yes_or_no = "[Y]".green + "Yes" + " [N]".red + "No"
+
+def validate_to_archive
+  answer_to_archive = gets.chomp
```

Comment:
🤔 Well, hopefully there is at least _one_ in the future, right? If I run the script and import an event, then run the script again, there will be one in the future. We could just compare w/ the date in the file name so we don't have to parse the frontmatter.